types/waf.go

```diff
@@ -97,31 +97,20 @@ func (re RuleEngineStatus) String() string {
 	return "unknown"
 }
 
-// RequestBodyLimitAction represents the action
+// BodyLimitAction represents the action
```

Comment:
Why not have request and response body limits?

Reply:
We have bodylimitaction for both so I created a unified type.